ts/components/MessagesTabIcon.tsx

```diff
@@ -1,38 +1,29 @@
 import React from "react";
 import { connect } from "react-redux";
 
-import { messagesUnreadAndUnarchivedSelector } from "../store/reducers/entities/messages";
-import { GlobalState } from "../store/reducers/types";
-import { usePaginatedMessages } from "../config";
 import TabIconComponent from "./ui/TabIconComponent";
 
 type OwnProps = {
   color?: string;
 };
 
-type Props = OwnProps & ReturnType<typeof mapStateToProps>;
+type Props = OwnProps;
 
 /**
  * Message tab icon with badge indicator
  */
 class MessagesTabIcon extends React.PureComponent<Props> {
   public render() {
-    const { color, messagesUnread } = this.props;
+    const { color } = this.props;
     return (
       <TabIconComponent
         iconName={"io-messaggi"}
         // badge is disabled with paginated messages see https://pagopa.atlassian.net/browse/IA-572
-        badgeValue={usePaginatedMessages ? undefined : messagesUnread}
+        badgeValue={undefined}
         color={color}
       />
     );
   }
 }
 
-function mapStateToProps(state: GlobalState) {
-  return {
-    messagesUnread: messagesUnreadAndUnarchivedSelector(state)
-  };
-}
-
-export default connect(mapStateToProps)(MessagesTabIcon);
+export default connect()(MessagesTabIcon);
```

Comment:
What do you think about removing the connect since the component doesn't need store connection anymore?

Reply:
Done! 8593756